### DISCUSSION TITLE
when a duplicate torrent is auto-added from a watchdir, 4.0.1-dev now gives a "duplicate torrent" notification

### DISCUSSION THREAD

**Discussion by dantearmok** [transmission/transmission#5145] [Support] [answered]
2.5-GHz Core i5 Mac mini (2012), 16GB RAM, plus an assortment of USB & FW HDs.
macOS Catalina, Safari, ClamXAV, and PIA (all fully updated).
Running Transmission 4.0.1-dev (7f9ef4730e) -- March 5th's nightly.

1. Drop a .torrent into ~/Downloads and Transmission auto loads it then gives the correct notification.

2. Drop a duplicate torrent and Transmission gives the same notification, instead of complaining about a duplicate.

3. Double-click on a duplicate .torrent file causes no response from Transmission.

I believe this is a regression from Transmission 3.0's behavior.  Donno if intentional or not.

- Dan.

**Comment by nevack** [comment]
Seems like correct behaviour.
Duplicated file is still new file with new name, so why not to try to add it?

**Comment by dantearmok** [comment]
By duplicate I mean two identical .torrent files.  Getting dups is usually the result of an error somewhere. 

Transmission 3 and earlier gave a proper response.  T4 is saying it auto-added the dup, when in fact it did nothing.  When you double-click a .torrent file, T4 is saying nothing when in fact it found the file to be a dup and didn't load it.

- Dan.

**Comment by ckerr** [comment]
@Coeur could this be related to 8a374a3310540f23500dbccf84a48609b7fea3c9 ?

**Comment by ckerr** [comment, marked as answer]
...I tried to make the subject line clearer, but it's not easy to write in a single sentence :smile: 

If a user manually tries to add a duplicate torrent, e.g. by dragging it into the app or via the open dialog, then a notification should be shown.

If a duplicate torrent is added _passively_ though, e.g. by a watchdir, there shouldn't be a notification.

**Comment by Coeur** [comment]
Sorry for my French screenshot. If you ever checked the checkbox, then you won't get the popup ever again, and the only way to get it back is in Transmission > Settings > General > Reset all warnings

![Capture d’écran 2023-03-10 à 01 53 07](https://user-images.githubusercontent.com/839992/224113986-2bcf76bf-f969-4ed8-9bb1-84f15025deb3.png)
